packages/design-system-twrnc-preset/tsconfig.build.json

```diff
@@ -8,6 +8,6 @@
     "types": ["react-native", "jest"],
     "emitDeclarationOnly": false
   },
-  "references": [],
+  "references": [{ "path": "../design-tokens/tsconfig.build.json" }],
```

Comment:
Add design-tokens to TypeScript project references in twrnc-preset

Adds design-tokens package as a TypeScript project reference in design-system-twrnc-preset's tsconfig.build.json. This ensures proper type checking and build ordering since twrnc-preset directly consumes design token values to generate the Tailwind configuration. The reference is needed to maintain TypeScript's understanding of the dependency relationship between these packages.

Note: This reference is only added to twrnc-preset (not react-native) as it's the package that directly consumes the design tokens, react native references the twrnc-preset